Removing github login (#6010)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/git/connect-github.md

   - Read and write access to Workflows

6. Once you grant access to the app, you will be redirected back to dbt Cloud and shown a linked account success state. You are now personally authenticated. 

> 7. Ask your team members to [personally authenticate](#authenticate-your-personal-github-account) by connecting their GitHub profiles.

Author: ```suggestion
7. Ask your team members to individually authenticate by connecting their [personal GitHub profiles](#authenticate-your-personal-github-account).
```

## Limiting repository access in GitHub
If you are your GitHub organization owner, you can also configure the dbt Cloud GitHub application to have access to only select repositories. This configuration must be done in GitHub, but we provide an easy link in dbt Cloud to start this process.
<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/configure-github.png" title="Configuring the dbt Cloud app"/>

> ## Authenticate your personal GitHub account

Reviewer: have links been checked since it's a heading change? iirc we need to do this for anchor links (ie links to sections on page) bc our link checker doesn't catch them

Author: Yes! There were two links that used the old header and I changed them. One on this page and one on another.

> ## Authenticate your personal GitHub account

Reviewer: ❤️ the new heading title, much better

> Once the dbt Cloud admin has [set up a connection](/docs/cloud/git/connect-github#installing-dbt-cloud-in-your-github-account) to your organization GitHub account, you need to personally authenticate.  You must connect your personal GitHub profile to dbt Cloud to use the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) and verify your read / write access to the repo.

Reviewer: ```suggestion
After the dbt Cloud administrator [sets up a connection](/docs/cloud/git/connect-github#installing-dbt-cloud-in-your-github-account) to your organization's GitHub account, you need to authenticate using your personal account. You must connect your personal GitHub profile to dbt Cloud to use the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) and verify your read and write access to the repository.
```

> After the dbt Cloud administrator [sets up a connection](/docs/cloud/git/connect-github#installing-dbt-cloud-in-your-github-account) to your organization's GitHub account, you need to authenticate using your personal account. You must connect your personal GitHub profile to dbt Cloud to use the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) and verify your read and write access to the repository.

Author: ```suggestion
After the dbt Cloud administrator [sets up a connection](/docs/cloud/git/connect-github#installing-dbt-cloud-in-your-github-account) to your organization's GitHub account, you need to authenticate using your personal account. You must connect your personal GitHub profile to dbt Cloud to use the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) and [CLI](/docs/cloud/cloud-cli-installation) and verify your read and write access to the repository.
```

4. Once you approve authorization, you will be redirected to dbt Cloud, and you should now see your connected account. 

> You're now ready to use the dbt Cloud IDE or dbt Cloud CLI.

Reviewer: ```suggestion
You can now use the dbt Cloud IDE or dbt Cloud CLI.
```

Reviewer: does it make sense to link out to the IDE and CLI pages here?

Author: Linked at the top of this section!
